### ISSUE TITLE
drawRoad not calculate/draw in web platfrom correctly

### ISSUE THREAD

**Issue by blazmahoric** [liodali/osm_flutter#509]
On web platform, the drawRoad method not showing or calculating foot road type correctly on some paths. On Android it is ok.

```
    // not showing or calculating correctly on web platform. It showing only dot.
    await controller.drawRoad(
      GeoPoint(latitude: 46.42481, longitude: 15.90403),
      GeoPoint(latitude: 46.42606, longitude: 15.90409),
      roadType: RoadType.foot,
      roadOption: const RoadOption(
          roadWidth: 15,
          roadColor: Colors.red,
          zoomInto: true,
          roadBorderWidth: 0),
    );

    // bottom coordinates works on both platforms.
    await controller.drawRoad(
      GeoPoint(latitude: 46.42388, longitude: 15.90294),
      GeoPoint(latitude: 46.42374, longitude: 15.90512),
      roadType: RoadType.foot,
      roadOption: const RoadOption(
          roadWidth: 10,
          roadColor: Colors.blueGrey,
          zoomInto: false,
          roadBorderWidth: 0),
    );
```

It looks like that method always calculating route for roadType.car on web platform. Altrough if we set foot or bike to roadType.

**Comment by liodali** [comment]
i will check the issue

**Comment by liodali** [comment]
which version you're currently use ?

**Comment by blazmahoric** [comment]
> which version you're currently use ?

Tested on both, 0.70.4 and 1.0.0-rc.5.

**Comment by liodali** [comment]
i will fix our dart-osrm-routing package thnx for report

**Comment by liodali** [comment]
i published a new version for rc `1.0.0-rc.6`

**Comment by blazmahoric** [comment]
Thanks!
